src/dashboard/src/media/js/vendor/base64.js

```diff
@@ -1,43 +0,0 @@
-// Adapted from https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64.btoa
```

Comment:
Sorry, that was unclear. You're symlinking instead of fixing the places that import it? Wouldn't it be easier long term to modify the imports instead of having the indirection?